### PR TITLE
Adding text to response body

### PR DESCRIPTION
**Pull request by jwineman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ following shape:
   headers: {...},
   status: 200,
   body: {...}
+  text: {...}
```
Comment (thejameskyle) [inline README.md]: Shouldn't this be a string?

Comment (jwineman) [inline README.md]: Yeah, whats convention to represent that in README? "string"?

Comment (thejameskyle) [inline README.md]: You can just write `'...'`

**Comment by thejameskyle** [comment]
btw, GitHub doesn't ping people when new commits are made (:cry:) so you have to make a comment.

**Pull request merged**